octobot_trading/modes/channel/abstract_mode_producer.py

```diff
@@ -455,20 +455,25 @@ async def _wait_for_symbol_init(self, symbol, time_frame, timeout) -> bool:
             self.logger.error(f"Initialization took more than {timeout} seconds")
         return False
 
-    async def _wait_for_bot_init(self, timeout, extra_topics: list=None) -> bool:
+    async def _wait_for_bot_init(self, timeout, extra_symbol_topics: list=None) -> bool:
         try:
-            topics = [
-                common_enums.InitializationEventExchangeTopics.BALANCE.value,
```

Comment:
I am not able to find BALANCE in the ouput diff, does BALANCE topic is removed?

Reply:
I moved it out of the symbol  for loop

Reply:
Oh yes my bad